daiquiri/__init__.py

```diff
@@ -88,8 +88,27 @@ def setup(level=logging.WARNING, outputs=[output.STDERR], program_name=None,
         root_logger.removeHandler(handler)
 
     # Add configured handlers
-    for o in outputs:
-        o.add_to_logger(root_logger)
+    for out in outputs:
+        # instantiating an output from a dict of options
+        if isinstance(out, dict):
```

Comment:
I'll leave that decision up to you.  I think it could be a win for people who want to read their conf from a JSON-ified source, but on the other hand, if they're doing that then probably they're talking to `logging` directly (since it can already do that iirc).